pkg/agent/datamodel/const.go

```diff
@@ -124,12 +124,14 @@ const (
 	Nvidia510GridDriverVersion = "grid-510.73.08"
 )
 
-// these sizes use a "converged" driver to support both cuda/grid workloads.
-// how do you figure this out? ask HPC or find out by trial and error.
-// installing vanilla cuda drivers will fail to install with opaque errors.
-// nvidia-bug-report.sh will is helpful, but usually it tells you the pci card id is incompatible.
-// That sends me to HPC folks.
-// see https://github.com/Azure/azhpc-extensions/blob/daaefd78df6f27012caf30f3b54c3bd6dc437652/NvidiaGPU/resources.json
+/* ConvergedGPUDriverSizes : these sizes use a "converged" driver to support both cuda/grid workloads.
+how do you figure this out? ask HPC or find out by trial and error.
+installing vanilla cuda drivers will fail to install with opaque errors.
+nvidia-bug-report.sh will is helpful, but usually it tells you the pci card id is incompatible.
```

Comment:
```suggestion
nvidia-bug-report.sh may be helpful, but usually it tells you the pci card id is incompatible.
```

former typo on myself